syntax = "proto2";

package recursive_dependency_embedding;

message SequenceNode {
  optional int64 head = 1;
  repeated SequenceNode children = 2;
}